src/bindings/pygaia/scripts/classification/train_model_from_sigs.py

```diff
@@ -87,19 +86,24 @@ def main(input_directory, output_directory, force=False, project_name=None):
     if os.path.exists(json_name):
         os.remove(json_name)
 
-    train_model.trainModel(groundtruth_name, yaml_name, project_file, resultsdir, results_model_file)
+    train_model.trainModel(groundtruth_name, yaml_name,
+                           project_file, resultsdir, results_model_file)
+
 
 if __name__ == "__main__":
-    parser = ArgumentParser()
-    # parser.description("Generates a model of the sig files in `directory`. Converts to yaml if needed.")
-    parser.add_argument('input_directory', help='directory with the json/sig files.')
-    parser.add_argument('output_directory', help='directory with the output files.')
-    parser.add_argument('--force', '-f', help='directory with the output files.', action='store_true')
-    parser.add_argument('--project_name', '-n', help='the project name.')
+    parser = ArgumentParser(description="Generates a model of the sig files "
```

Comment:
We can improve the description a little bit. 

"Generates a model trained using descriptor files (*.sig, *.json) in a  `directory`. Each subdirectory is considered a class with the descriptor files therein being its training examples. JSON files are converted to YAML if needed."